data_models/website.py

```diff
@@ -0,0 +1,38 @@
+"""This script creates a class to store
+information about to a website"""
+
+import validators
+
+from data_models.contents import Contents
+
+
+class Website:
+    """This class stores the information about a website"""
+
+    def __init__(self, url):
+        self.url = url
+        self.is_valid = self.__validate_url()
+        self.domain = self.__get_domain()
+        self.logo_url = self.__extract_logo_url()
```

Comment:
I have set these attributes inside constructor. Let me know if this is wrong practice.

Reply:
Okay. Sounds good. I will do it.

Reply:
They are associated with conditions so I think having a function would be better. Let me know your thoughts.

Reply:
validators.url(self.url) returns True if valid otherwise it returns ValidationFailure() instead of False. So we might need to add `self.is_valid = validators.url(self.url) is True`. This will be one liner so we can assign it directly.

Reply:
I meant, you can convert it into one liners using ternary operator if needed.

Reply:
https://softwareengineering.stackexchange.com/questions/107669/one-line-functions-that-are-called-only-once?noredirect=1&lq=1

Reply:
Updated accordingly. Let me know if any further changes are required.